src/aiortc/contrib/media.py

```diff
@@ -463,6 +463,17 @@ def addTrack(self, track):
             stream = None
         self.__tracks[track] = MediaRecorderContext(stream)
 
+    def setsize(self, track):
```

Comment:
please use __set_size since it is a helper function (seems to be the file convention)

Reply:
Also what was the observed bug that prompted this?

Reply:
The `track` responsible to generate the video is different for the case when enable_prediction is True or when it's False.